db/Pos/DB/Pure.hs

```diff
@@ -94,6 +95,9 @@ type MonadPureDB ctx m =
     , MonadIO m
     )
 
+dbPureDump :: MonadPureDB ctx m => m DBPure
```

Comment:
It would be good to use it in `VarSpec` already (there this functionality is inlined).